docs/pages/04_types.adoc

```diff
@@ -484,26 +484,31 @@ Time literals can be represented as literals using the following syntax.
 
 [subs="quotes"]
 ----
-[purple]##**\`**##&lt;hh>[purple]##**:**##&lt;mm>&#x5B;[purple]##**:**##&lt;ss>][purple]##**`**##
+[purple]##**\`**##&lt;hh>[purple]##**:**##&lt;mm>&#x5B;[purple]##**:**##&lt;ss>[.&lt;ff>]][purple]##**`**##
 ----
 
 where
 
 * <hh> refers to a zero-padded hour between 00 and 23,
 * <mm> refers to a zero-padded minute between 00 and 59,
 * <ss> refers to a zero-padded second between 00 and 59,
+* <ff> refers to fractional second (millisecond) between 000 and 999 (trailing zeros can be omitted: `12:23:56.100 = 12:23:56.1`),
 * the surrounding backticks are required.
 
-The valid values of time are between `pass:[`00:00:00`]` and `pass:[`23:59:59`]`.
+The valid values of time are between `pass:[`00:00:00.000`]` (or `pass:[`00:00`]`) and `pass:[`23:59:59.999`]`.
```

Comment:
why is this necessary?

Reply:
I updated the documentation to align with what the backend can handle and JSL syntax allows. However some of these capabilities can be hidden. TBD tomorrow